extensions/linkedin.md

```diff
@@ -0,0 +1,29 @@
+# Linkedin Extension
+This document defines how to use `linkedin` extension in AsyncAPI documents.
+
+## Overview 
+This extension allows you to provide the Linkedin username of the account representing the team/company of the API.
+
+## Version
+Current version is `0.1.0`.
+
+## Extension Definition
+
+### Type: String
+
+Name of the Linkedin username.
```

Comment:
now, technically in case of linkedin please check, I think that they usually do not promote usernames but full profile links

look at https://www.linkedin.com/help/linkedin/answer/a522735

in case of projects like ours it is https://www.linkedin.com/company/asyncapi and in case of humans link is different. So I think we should expect full profile URL - as from the username, you do not know how later in the backed build the url to the profile

Reply:
yeahh added this 
`URL of the Linkedin Profile of the user or company.`